### DISCUSSION TITLE
DeluxeMusic

### DISCUSSION THREAD

**Discussion by Gofunk** [iptv-org/community#70] [Q&A] [answered]
Why isn't DeluxeMusic included in the music list? 
It's available in streaming here https://deluxemusic.de/

Do not confuse with Deluxe Lounge (which is listed already).

**Comment by BellezaEmporium** [comment, marked as answer]
We simply didn't know this existed. Here's a link to the stream : https://sdn-global-live-streaming-packager-cache.3qsdn.com/13456/13456_264_live.m3u8

**Comment by Gofunk** [comment]
ok, good to know. Thanks
